test/integration/password_reset_test.rb

```diff
@@ -1,19 +1,25 @@
 require 'test_helper'
+include ActiveJob::TestHelper
 
 class PasswordResetTest < SystemTest
   setup do
     @user = create(:user, handle: nil)
+    @old_queue_adapter = queue_adapter
+    ActiveJob::Base.queue_adapter = :test
```

Comment:
Isn't job being executed only if it is enqueued in `perform_enqueued_jobs` block?